src/main/resources/compendium.yaml

```diff
@@ -0,0 +1,250 @@
+openapi: 3.0.0
+info:
+  description: Compendium is a service for schema storage, conversion, version management,
+    and client generation. It supports a RESTful interface for storing and
+    serving schemas like Avro, Protobuf, OpenAPI, and Mu.
+  version: 0.0.1
+  title: Compendium
+  contact:
+    email: contact@higherkindness.io
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+externalDocs:
+  description: Learn more about Compendium
+  url: https://github.com/higherkindness/compendium
+servers:
+  - url: '{protocol}://{host}:{port}/{urlPrefix}'
+    variables:
+      protocol:
+        enum:
+          - http
+          - https
+        default: http
+      host:
+        default: localhost
+      port:
+        default: "8080"
+      urlPrefix:
+        default: v0
+tags:
+  - name: health
+    description: health ping
+  - name: protocol
+    description: Operations on protocols
+    externalDocs:
+      description: Learn more
+      url: https://github.com/higherkindness/compendium
+paths:
+  "/health":
+    get:
+      servers:
+        - url: '{protocol}://{host}:{port}'
+          description: Override base path for health response endpoint
+          variables:
+            protocol:
+              default: http
+            host:
+              default: localhost
+            port:
+              default: "8080"
+      tags:
+        - health
+      summary: health response endpoint
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/HealthResponse"
+          description: Service is healthy
+        "400":
+          description: Bad request
+        "404":
+          description: Not found
+        "500":
+          description: Internal server error
+  "/protocol/{protocolId}":
+    post:
+      tags:
+        - protocol
+      summary: store a protocol
+      parameters:
+        - name: protocolId
+          in: path
+          required: true
+          description: the id of the protocol - can be a string of any alphanumeric
+            character, dash (-), or period (.) with a length less than or equal to 255
+            characters
```

Comment:
```suggestion
          description: the id of the protocol
```

Reply:
I think this is correct, is a good specification of what compendium expects

Reply:
The type has a regex expression checking it